shared-doc/view-the-source.adoc

```diff
@@ -1,3 +1,3 @@
 ifndef::ProductRelease,EAPXPRelease[]
-link:https://github.com/wildfly/quickstart/tree/{artifactVersion}/{artifactId}[Browse the source]
+link:https://github.com/wildfly/quickstart/tree/{WildFlyQuickStartRepoTag}/{artifactId}[Browse the source]
```

Comment:
Ah, if I'd known about this there'd not have been a need for the parent upgrade. :)